src/sdk.ts

```diff
@@ -497,6 +497,30 @@ export class DefaultRelayingServices implements RelayingServices {
         );
     }
 
+    async validateSmartWallet(address: string): Promise<SmartWallet> {
+        log.debug('Validating smart wallet', address);
+
+        log.debug('Checking if the wallet already exists');
+        const isSmartWalletDeployed = await this.isSmartWalletDeployed(address);
+
+        if (!isSmartWalletDeployed) {
+            throw new Error('Smart Wallet  is not deployed');
+        }
+
+        const txDetails: EnvelopingTransactionDetails = {
+            from: this._getAccountAddress(),
+            to: ZERO_ADDRESS,
+            callForwarder: address,
+            data: '0x'
+        };
+        await this.relayProvider.relayClient.validateSmartWallet(txDetails);
+        return {
+            index: -1,
```

Comment:
What's the meaning of -1 in this context? It this a good place to create a descriptive constant? Somthing like index: SOME_DESCRIPTION_OF_THE_CONSTANT